tests/bot/cogs/moderation/test_utils.py

```diff
@@ -212,54 +212,31 @@ def setUp(self):
 
     async def test_post_user(self):
         """Test does `post_user` handle errors and results correctly."""
+        user = MockUser(avatar="abc", discriminator=5678, id=1234, name="Test user")
         test_cases = [
             {
-                "args": (self.ctx, self.user),
-                "post_result": [
-                    {
-                        "id": 1234,
-                        "avatar": "test",
-                        "name": "Test",
-                        "discriminator": 1234,
-                        "roles": [
-                            1234,
-                            5678
-                        ],
-                        "in_guild": False
-                    }
-                ],
+                "args": (self.ctx, user),
+                "post_result": "bar",
```

Comment:
You can just "hard-code" these in the test code, no need to define different values in the test cases.